cmd/fleet/cron.go

```diff
@@ -225,6 +227,10 @@ func cronVulnerabilities(
 			level.Debug(logger).Log("vulnAutomationEnabled", vulnAutomationEnabled)
 
 			recentVulns := checkVulnerabilities(ctx, ds, logger, vulnPath, config, (vulnAutomationEnabled != ""))
+
+			// TODO: merge results
+			checkOvalVulnerabilities(ctx, ds, logger, vulnPath, config)
```

Comment:
(Am still reviewing) This means Fleet server will need to connect to `security-metadata.canonical.com`?

Let's check if we are ok with this.

The other option was for the NVD async job to do this fetching and pre-processing and expose the info in the NVD release tar.gz.

Reply:
Yes, not only `security-metadata.canonical.com` but also `www.redhat.com/security/data/oval/` once we implement the CentOS functionality. 

As a reminder, we download an oval file per platform per version.

Reply:
We just need to define the direction we want to move forward:
1. We are ok with fleet server connecting to a specific/defined list of external hosts (ever growing list as we move forward with more ingestion of different data: OVAL, first.org, CISA, etc.).
2. We do all the "external" communication in NVD repository and fleet server only connects to Github+NVD (we could even move the nvd.nist.gov ingestion to NVD repo in the future).

Maybe (1) is a-ok, but we need to agree it won't cause issues/burden to existing customers/infrastructure.

Reply:
The team decided to go with 1 for the time being, I'll mark this as resolved.

Reply:
As discussed on the Slack channel, let's document which hosts does Fleet need to connect to.

We could start a new section "external hosts" or something like that on:
https://github.com/fleetdm/fleet/blob/main/docs/Using-Fleet/Vulnerability-Processing.md